jmix-flowui/flowui-kit/src/main/resources/META-INF/frontend/src/timer/jmix-timer.js

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright 2023 Haulmont.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { html, PolymerElement } from '@polymer/polymer';
+import { ElementMixin } from '@vaadin/component-base/src/element-mixin.js';
+
+export class JmixTimer extends ElementMixin(PolymerElement) {
```

Comment:
make it hidden. Not it produces extra spacing at it has its own element in DOM tree